docs/Customizing Look and Functionality/Customizing Help Prompts/index.md

```diff
@@ -0,0 +1,184 @@
+# Customizing Help Prompts
+On some head units it is possible to display a customized help menu or speak a custom command if the user asks for help while using your app. The help menu is commonly used to let users know what voice commands are available however it can also be customized to help your user navigate the app or let them know what features are available. 
```

Comment:
```suggestion
On some head units it is possible to display a customized help menu or speak a custom command if the user asks for help while using your app. The help menu is commonly used to let users know what voice commands are available, however, it can also be customized to help your user navigate the app or let them know what features are available. 
```